scene/gui/scroll_container.cpp

```diff
@@ -239,13 +239,13 @@ void ScrollContainer::_update_scrollbar_position() {
 }
 
 void ScrollContainer::_gui_focus_changed(Control *p_control) {
-	if (follow_focus && is_a_parent_of(p_control)) {
+	if (follow_focus && is_ancestor_of(p_control)) {
 		ensure_control_visible(p_control);
 	}
 }
 
 void ScrollContainer::ensure_control_visible(Control *p_control) {
-	ERR_FAIL_COND_MSG(!is_a_parent_of(p_control), "Must be a parent of the control.");
+	ERR_FAIL_COND_MSG(!is_ancestor_of(p_control), "Must be a parent of the control.");
```

Comment:
```suggestion
	ERR_FAIL_COND_MSG(!is_ancestor_of(p_control), "Must be an ancestor of the control.");
```

Maybe the text message should be changed to "ancestor" too?